src/components/CarbonaraCalculator.js

```diff
@@ -690,15 +693,18 @@ class CarbonaraCalculator extends Component {
 
                             </div>
                             { showGamificationResults &&
-                                <div uk-grid="" className="gamification-result uk-grid-collapse uk-light">
-                                    <div className="uk-width-1-1@s">
-                                        <strong>{Math.round(this.state.gamificationResult)} kg</strong> CO<sub>2</sub> emissions correspond to:
-                                    </div>
-                                    <div className="uk-width-1-1@s">
-                                        <strong>{gamificationKilometersDrivenByCar}&nbsp;km</strong>&nbsp;driven by&nbsp;car
-                                    </div>
-                                    <div className="uk-width-1-1@s">
-                                        <strong>{gamificationYearsOfCo2Sequestration}&nbsp;years</strong>&nbsp;of CO<sub>2</sub> sequestration
+                                <div className="uk-width-1-1 uk-inline">
+                                    <div uk-grid="" className="gamification-result uk-grid-collapse uk-light">
+                                        <div className="uk-width-1-1@s">
+                                            <strong>{Math.round(this.state.gamificationResult)} kg</strong> CO<sub>2</sub> emissions correspond to:
+                                        </div>
+                                        <div className="uk-width-1-1@s">
+                                            <strong>{gamificationKilometersDrivenByCar}&nbsp;km</strong>&nbsp;driven by&nbsp;car
+                                        </div>
+                                        <div className="uk-width-1-1@s">
+                                            <strong>{gamificationYearsOfCo2Sequestration}&nbsp;years</strong>&nbsp;of CO<sub>2</sub> sequestration
+                                        </div>
+                                        <span uk-icon="question" uk-tooltip="Calculations are based on several assumptions, therefore the underlying data and calculation results are only approximations, and could change as more information is gathered." className="uk-text-center uk-padding-small uk-position-center-right"></span>
```

Comment:
Tooltips are displayed on hover, which is not possible on mobile devices.

Reply:
I tried it on my phone and the tooltip appears upon clicking on the icon. This might be something that UIkit take cares of?

Reply:
That's cool. Didn't know that 👍